Source/Blazorise/Utilities/IO/IFileEntryOwner.cs

```diff
@@ -44,10 +45,12 @@ public interface IFileEntryOwner
     /// <summary>
     /// Element reference.
     /// </summary>
+    [JsonIgnore]
```

Comment:
I believe ElementRef should be able to be serialized. ElementReference is a known Blazor related type.

Reply:
Thank you!